src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Added ability to export attributes corresponding to `LogRecord.Exception` i.e.
+`exception.type`, `exception.message` and `exception.stacktrace`. These
+attributes will be exported when
+`OTEL_DOTNET_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES` environment variable will be
+set to `true`.
+([#4892](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4892))
```

Comment:
We should move this below the other 2 "unreleased" entries right? I've been telling people recently to order things chronologically hopefully I'm not mistaken 🤣

Reply:
we do most recent one at the top
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#unreleased

Reply:
I vote for 1) the latest release should be at the top, the oldest release should be at the bottom 2) within a release, changes are ordered based on when the PR got merged (the latest one should be at the bottom).

Example for 1) https://github.com/dotnet/core/tree/main/release-notes/8.0#releases

Example for 2) https://github.com/dotnet/core/blob/main/release-notes/8.0/preview/8.0.0-rc.1.md#notable-fixes

Reply:
@vishweshbankwar The unreleased section is at the top, yes, but the new entry goes into the bottom of that section 😄

Reply:
I need to get my eyes checked😆 . Fixed.